src/language/concurrency/index.md

```diff
@@ -314,17 +314,16 @@ setting up and managing worker isolates:
 
 {{site.alert.flutter-note}}
   If you're using Flutter,
-  consider using [Flutter's `compute()` function][]
+  you can use [Flutter's `compute` function][]
   instead of `Isolate.run()`.
-  The `compute` function
-  allows your code to work  on both
-  [native and non-native platforms][].
+  On the [web](#web), the `compute` function falls back
+  to running the specified function on the current event loop.
   Use `Isolate.run()` when targeting native platforms only
```

Comment:
Unrelated but could you toss a comma in there for me:
```suggestion
  Use `Isolate.run()` when targeting native platforms only,
```

Reply:
Just for you!